File: VB/WpfReportDesigner_local/LocalReportWizardServiceClientFactory.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.Data.XtraReports.ServiceModel

Namespace WpfReportDesigner_local
	Friend Class LocalReportWizardServiceClientFactory
		Inherits ReportWizardServiceClientFactory
		Protected Overrides Function CreateClient(ByVal channel As IAsyncReportWizardService) As IReportWizardServiceClient
			Return New LocalReportWizardServiceClient()
		End Function
	End Class
End Namespace

File: CS/WpfReportDesigner_local/LocalReportWizardServiceClientFactory.cs
using DevExpress.Data.XtraReports.ServiceModel;

namespace WpfReportDesigner_local {
    class LocalReportWizardServiceClientFactory : ReportWizardServiceClientFactory {
        protected override IReportWizardServiceClient CreateClient(IAsyncReportWizardService channel) {
            return new LocalReportWizardServiceClient();
        }
    }
}
